### PR TITLE
feat: add referral bonus stats

### PR DESCRIPTION
**Pull request by jpangelle**
### Description

This pull request adds the stats dashboard to the staking page. Also adds the total contributions in the header

[Preview Link](https://6153d016962ced525fe64d99--centrifuge-website.netlify.app/altair/crowdloan)

##### Screenshots
image

image

### CODE REVIEW

**Review by armandomistral — APPROVED** [review]
Looks good to me, @jpangelle!! Thanks

**Review by onnovisser — APPROVED** [review]
👍

File: package.json [hunk 1]
```diff
@@ -12,11 +12,14 @@
     "@mdx-js/react": "^1.6.1",
     "@polkadot/api": "^4.16.2",
     "@polkadot/extension-dapp": "^0.38.7",
+    "@polkadot/util": "^7.3.1",
     "@polkadot/util-crypto": "^6.10.1",
     "@researchgate/react-intersection-observer": "0.7.4",
     "@sendgrid/mail": "^7.1.1",
     "abort-controller": "^3.0.0",
+    "axios": "^0.21.4",
```
Comment (alessioscalici) [inline package.json:20]: Is there a reason to use `axios` instead of standard `fetch`?

File: package.json [hunk 2]
```diff
@@ -12,11 +12,14 @@
...
     "babel-plugin-styled-components": "1.10.0",
+    "bignumber.js": "^9.0.1",
```
Comment (alessioscalici) [inline package.json:22]: Probably silly question: is it necessary to use BigNumber here? Wouldn' be enough using native `number` type, considering it's for presentation?

**Review by alessioscalici — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (alessioscalici) [reply package.json:20]: resolving, I didn't notice this is only used in lambdas 😅

File: package.json (hunk 2 above)
Comment (alessioscalici) [reply package.json:22]: Cool, thanks for the clarification! resolving!

**Review by alessioscalici — APPROVED** [review]

**Review by armandomistral — APPROVED** [review]

**Review by alessioscalici — APPROVED** [review]

**Pull request closed without merging**